assets/stylesheets/test-details.scss

```
@@ -414,4 +414,14 @@ ul.modcategory {
     .fa-download:hover {
         opacity: inherit;
     }
+}
+
+.test-result-overview-link {
+    top: -0.1em;
+    position: relative;
+    left: 0.2em;
+    opacity: 0.8;
+    font-size:medium;
```

Don't set the size to a possibly user-defined default. It might not fit because in all other places we don't use such values. Just make it relative to the size of the heading:

```suggestion
    font-size: font-size: 75%;;
```

Also mind the coding style (the space was missing).

---

done